commercetools-models/src/main/java/io/sphere/sdk/orders/messages/LikeLineItemStateTranstition.java

```diff
@@ -0,0 +1,41 @@
+package io.sphere.sdk.orders.messages;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.sphere.sdk.messages.GenericMessageImpl;
+import io.sphere.sdk.models.Reference;
+import io.sphere.sdk.orders.Order;
+import io.sphere.sdk.states.State;
+
+import java.time.ZonedDateTime;
+
+public class LikeLineItemStateTranstition extends GenericMessageImpl<Order> {
+
+    protected final ZonedDateTime transitionDate;
+    protected final Long quantity;
+    protected final Reference<State> fromState;
+    protected final Reference<State> toState;
+
+    public LikeLineItemStateTranstition(final String id, final Long version, final ZonedDateTime createdAt, final ZonedDateTime lastModifiedAt, final JsonNode resource, final Long sequenceNumber, final Long resourceVersion, final String type, final Class<Order> clazz, final ZonedDateTime transitionDate, final Long quantity, final Reference<State> fromState, final Reference<State> toState) {
```

Comment:
I also though of having this class `abstract` but `GenericMessageImpl` is not `abstract`